### ISSUE TITLE
AttributeError: module 'sipbuild.api' has no attribute 'prepare_metadata_for_build_wheel'

### ISSUE THREAD

**Issue by jdeisenberg** [srevinsaju/guiscrcpy#101]
```
~> python3 -m pip install guiscrcpy
Collecting guiscrcpy
  Using cached https://files.pythonhosted.org/packages/0f/ab/31b59a5d862331c4352c3b5b4731873c7019ac3f0a0bd05e4054133c545c/guiscrcpy-3.7-py3-none-any.whl
Collecting psutil (from guiscrcpy)
  Using cached https://files.pythonhosted.org/packages/c4/b8/3512f0e93e0db23a71d82485ba256071ebef99b227351f0f5540f744af41/psutil-5.7.0.tar.gz
Collecting PyQt5==5.14.2 (from guiscrcpy)
  Using cached https://files.pythonhosted.org/packages/4d/81/b9a66a28fb9a7bbeb60e266f06ebc4703e7e42b99e3609bf1b58ddd232b9/PyQt5-5.14.2.tar.gz
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    Complete output from command /usr/bin/python3 /usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpsw4dezl8:
    Traceback (most recent call last):
      File "/usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 64, in prepare_metadata_for_build_wheel
        hook = backend.prepare_metadata_for_build_wheel
    AttributeError: module 'sipbuild.api' has no attribute 'prepare_metadata_for_build_wheel'
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 207, in <module>
        main()
      File "/usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 197, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "/usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 67, in prepare_metadata_for_build_wheel
        config_settings)
      File "/usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 95, in _get_wheel_metadata_from_wheel
        whl_basename = backend.build_wheel(metadata_directory, config_settings)
      File "/tmp/pip-build-env-c4_avjto/overlay/lib64/python3.7/site-packages/sipbuild/api.py", line 51, in build_wheel
        project = AbstractProject.bootstrap('pep517')
      File "/tmp/pip-build-env-c4_avjto/overlay/lib64/python3.7/site-packages/sipbuild/abstract_project.py", line 82, in bootstrap
        project.setup(pyproject, tool, tool_description)
      File "/tmp/pip-build-env-c4_avjto/overlay/lib64/python3.7/site-packages/sipbuild/project.py", line 410, in setup
        self.apply_user_defaults(tool)
      File "project.py", line 62, in apply_user_defaults
        super().apply_user_defaults(tool)
      File "/tmp/pip-build-env-c4_avjto/overlay/lib/python3.7/site-packages/pyqtbuild/project.py", line 86, in apply_user_defaults
        super().apply_user_defaults(tool)
      File "/tmp/pip-build-env-c4_avjto/overlay/lib64/python3.7/site-packages/sipbuild/project.py", line 215, in apply_user_defaults
        self.builder.apply_user_defaults(tool)
      File "/tmp/pip-build-env-c4_avjto/overlay/lib/python3.7/site-packages/pyqtbuild/builder.py", line 68, in apply_user_defaults
        "specify a working qmake or add it to PATH")
    sipbuild.pyproject.PyProjectOptionException
    
    ----------------------------------------
Command "/usr/bin/python3 /usr/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpsw4dezl8" failed with error code 1 in /tmp/pip-install-knzxjiek/PyQt5
```

**Comment by jdeisenberg** [comment]
Upgrading pip3 to the latest version (20.0.2) appears to have solved the problem.

**Comment by xjcl** [comment]
Upgrading pip did not solve this for me

**Comment by srevinsaju** [comment]
@xjcl also depends on the python version you are having. Guiscrcpy needs the latest python version

**Comment by xjcl** [comment]
I'm actually trying to install pyqt6, but Google led me here. I tried with Python 3.7 and 3.9, both with old and current versions of pip.

**Comment by srevinsaju** [comment]
Cool, actually, PyQt6 was released on 4 January 2021. Its been only a week since its release. And guiscrcpy doesnt support Qt6 yet, but will surely support soon. Please create another issue for Qt6 support, I will more than glad to move the conversation there, and add it to the v5.0.0 milestone!. Thanks!

**Comment by foxtrotdev** [comment]
Here is the list of commands I had to use to successfully install guiscrcpy on a Mac.

```
curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
python get-pip.py

pip3 install --upgrade pip
pip3 install pyqt5
pip3 install guiscrcpy
```

**Comment by xjcl** [comment]
@srevinsaju Thanks for your reply. Sadly I am not really able to invest any time into this as I was just helping out a friend who used pyqt6 and have no experience with or interest in Qt at the moment.

The result is that I was able to pip install pyqt6 on Windows 10, but not on Linux Mint 18.3 (I tried with both Python 3.5, 3.7, and 3.9) -- maybe it's a Linux issue, maybe it's an issue with me running an old distro that is 2 months from end-of-life. I just wanted to put this information somewhere online in case it can be useful to anyone.
